integration-tests/README.md

```diff
@@ -1,3 +1,12 @@
-# integration-tests Project
+# integration-tests
 
-WIP
\ No newline at end of file
+Module contains integration tests verifying end to end functionality in real cluster. The tests are executed using command `mvn clean verify -Pcucumber -Devent-bridge.manager.url="" -Dkey-cloak.url=""'.
```

Comment:
Please add some example values for URLs in readme.
For example Keycloak URL for a Minikube needs to be in form http://localhost:30007/auth/realms/event-bridge-fm

Reply:
Sure